src/rgw/driver/sfs/sqlite/sqlite_versioned_objects.cc

```diff
@@ -121,4 +121,28 @@ SQLiteVersionedObjects::get_last_versioned_object(const uuid_d& object_id
   return ret_value;
 }
 
+std::vector<DBOPVersionedObjectInfo>
+SQLiteVersionedObjects::get_deleted_versioned_objects_desc(uint max_objects
```

Comment:
"desc" on its own has little meaning. Perhaps "_highest_priority_first" to tell callers that they should start with the first?

Reply:
done